src/components/dex/Trade.vue

```diff
@@ -0,0 +1,523 @@
+        <div class="value">{{ $formatNumber(baseHolding.totalBalance) }}</div>
+      </div>
+      <div @click="actionableHolding = aphHolding" :class="['balance', {active: aphHolding.symbol === actionableHolding.symbol}]" :title="aphBalanceToolTip" v-if="baseHolding.symbol !== 'APH' && quoteHolding.symbol !== 'APH'">
+        <div class="label">Balance (APH)</div>
+        <div class="value">{{ $formatNumber(aphHolding.totalBalance) }}</div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import { BigNumber } from 'bignumber.js';
+import MarketPairChart from './MarketPairChart';
+import MarketSelector from './MarketSelector';
+
+const ORDER_TYPES_LIST = [
+  {
+    label: 'Limit',
+    value: 'Limit',
+  },
+];
+
+export default {
+  components: {
+    MarketPairChart,
+    MarketSelector,
+  },
+
+  computed: {
+    amountLabel() {
+      if (!this.currentMarket) {
+        return '';
+      }
+
+      return 'ATI'; // this.$t('amountQuote', { quote: this.currentMarket.quoteCurrency });
+    },
+
+    aphBalanceToolTip() {
+      return 'walletBalanceContractBalance'; // TODO: Fix this
+    },
+
+    aphHolding() {
+      if (this.currentMarket && this.$store.state.holdings) {
+        const holding = _.find(this.$store.state.holdings, (o) => {
+          return o.assetId === this.$services.assets.APH;
+        });
+
+        if (holding) {
+          return holding;
+        }
+      }
+
+      return {
+        symbol: 'APH',
+        balance: 0,
+        totalBalance: 0,
+        contractBalance: 0,
+        openOrdersBalance: 0,
+      };
+    },
+
+    baseBalanceToolTip() {
+      return 'walletBalanceContractBalance'; // TODO: Fix this
+    },
+
+    baseHolding() {
+      if (this.currentMarket && this.$store.state.holdings) {
+        const holding = _.find(this.$store.state.holdings, (o) => {
+          return o.assetId === this.currentMarket.baseAssetId;
+        });
+
+        if (holding) {
+          return holding;
+        }
+      }
+
+      return {
+        symbol: this.currentMarket ? this.currentMarket.baseCurrency : '',
+        balance: 0,
+        totalBalance: 0,
+        contractBalance: 0,
+      };
+    },
+
+    canPlaceMarketOrder() {
+      const currentWallet = this.$services.wallets.getCurrentWallet();
+      return currentWallet && currentWallet.isLedger !== true;
+    },
+
+    currentMarket() {
+      return this.$store.state.currentMarket;
+    },
+
+    estimate() {
+      const holding = this.currentMarket ?
+        this.$services.neo.getHolding(this.currentMarket.baseAssetId).unitValue :
+        0;
+
+      console.log('holding', holding);
+
+      try {
+        return new BigNumber(this.total).multipliedBy(
+          new BigNumber(holding));
+      } catch (e) {
+        console.log(e);
+        return 0;
+      }
+    },
+
+    isMarketClosed() {
+      return this.$store.state.currentMarket && this.$store.state.currentMarket.isOpen === false;
```

Comment:
`return this.$store.state.currentMarket && !this.$store.state.currentMarket.isOpen`